build.sbt

```diff
@@ -7,27 +7,13 @@ import de.heikoseeberger.sbtheader.license.Apache2_0
 lazy val `akka-persistence-query-view` = (project in file("."))
   .enablePlugins(GitVersioning, GitBranchPrompt, BuildInfoPlugin)
   .settings(
-    organization := "com.ovoenergy",
-    organizationHomepage := Some(url("https://www.ovoenergy.com/")),
+    organization := "com.firstbird",
+    organizationHomepage := Some(url("https://www.firstbird.com")),
     description := "An Akka PersistentView replacement",
     name := "akka-persistence-query-view",
-    homepage := Some(url("https://github.com/ovotech/akka-persistence-query-view")),
+    homepage := Some(url("https://github.com/firstbirdtech/akka-persistence-query-view")),
     startYear := Some(2016),
     licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
-    developers := List(
```

Comment:
please add proper information and not just remove them, i think this is also necessary for bintray publishing (at least it is good practice and required if you publish anything under apache v2)